### PR TITLE
Trello card: Added Kubernetes Storage Metrics via Prometheus section

### PR DESCRIPTION
**Pull request by ghost**
Trello card: https://trello.com/c/J58JYpOl/714-document-cm-ops-tools-storage-prometheus-endpoint-coverage

### CODE REVIEW

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -1031,6 +1035,254 @@ Once `openshift_metrics_project: openshift-infra` is installed, metrics can be
...
+persistentvolumeclaim=<persistentvolumeclaim-name> 
+persistentvolume=<persistentvolume-name>
+|===
+
+[[k8s-desired-metrics]]
+==== Desired Metrics
+This section describes the desired metrics to be emitted from Kubernete's storage subsystem. These metrics are not currently emitted by the system, but 
+they can help to identify the health of the storage subsystem and cluster.
+
+.Desired Metrics
```
Comment (gnufied) [inline install_config/cluster_metrics.adoc]: There is no need to add Desired metrics section here. It was present in original Google doc for data collection.

File: install_config/cluster_metrics.adoc [hunk 2]
```diff
@@ -1031,6 +1035,254 @@ Once `openshift_metrics_project: openshift-infra` is installed, metrics can be
 gathered from the `http://${POD_IP}:7575/metrics` endpoint.
 ====
 
+[[openshift-prometheus-kubernetes-metrics]]
+=== Kubernetes Storage Metrics via Prometheus
```
Comment (gnufied) [inline install_config/cluster_metrics.adoc]: I think we should use word {{product-title}} rather than hardcoding Kubernetes or Openshift everywhere

Comment (tmorriso-rh) [reply install_config/cluster_metrics.adoc]: We don't have an attribute for Kubernetes. The {product-title} attribute is only defined for OpenShift.

**Comment by tmorriso-rh** [comment]
[rev_history]
|xref:../install_config/cluster_metrics.adoc#openshift-prometheus[Enabling Cluster Metrics -> Prometheus on OpenShift Container Platform]
|Added the xref:../install_config/cluster_metrics.adoc#openshift-prometheus-kubernetes-metrics[Kubernetes Storage Metrics via Prometheus] section.
%

**Comment by tmorriso-rh** [comment]
[rev_history]
|xref:../install_config/cluster_metrics.adoc#openshift-prometheus[Enabling Cluster Metrics -> Prometheus on OpenShift Container Platform]
|Added the xref:../install_config/cluster_metrics.adoc#openshift-prometheus-kubernetes-metrics[Kubernetes Storage Metrics via Prometheus] section.
%

File: install_config/cluster_metrics.adoc (hunk 2 above)
Comment (gnufied) [reply install_config/cluster_metrics.adoc]: I meant renamed "Kubernetes Storage metrics" to "Openshift storage metrics"

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: install_config/cluster_metrics.adoc [hunk 3]
```diff
@@ -37,6 +37,10 @@ each node individually through the `/stats` endpoint. From there, Heapster
 scrapes the metrics for CPU, memory and network usage, then exports them into
 Hawkular Metrics.
 
+The storage volume metrics available on the kubelet are not available through
+the `/stats` endpoint, but are available through the ``/metrics` endpoint. See
```
Comment (bfallonf) [inline install_config/cluster_metrics.adoc]: double backtick at the start of the /metrics bit?

File: install_config/cluster_metrics.adoc [hunk 4]
```diff
@@ -1031,6 +1034,118 @@ Once `openshift_metrics_project: openshift-infra` is installed, metrics can be
...
+link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cloud-provider/cloudprovider-storage-metrics.md[Cloud
+Provider (specifically GCE and AWS) metrics for Storage API calls] for more
+information.
+
+*Volume Operation Metrics*
+
+These metrics report time taken by a storage operation once started. These
+metrics keep track of operation time at the plug-in level, but do not include
+time taken by `goroutine` to run or operation to be picked up from the internal
+queue. These metrics are of type histogram.
```
Comment (bfallonf) [inline install_config/cluster_metrics.adoc]: Histogram is in ""s above. Should this one too?

**Comment by bfallonf** [comment]
@tmorriso-rh A couple comments. LG otherwise :railway_car:

File: install_config/cluster_metrics.adoc (hunk 3 above)
Comment (tmorriso-rh) [reply install_config/cluster_metrics.adoc]: Fixed.

File: install_config/cluster_metrics.adoc (hunk 4 above)
Comment (tmorriso-rh) [reply install_config/cluster_metrics.adoc]: Fixed.

**Pull request merged**